app/models/manager_refresh/inventory_collection_default/infra_manager.rb

```diff
@@ -82,17 +118,61 @@ def custom_attributes(extra_attributes = {})
       attributes = {
         :model_class => ::CustomAttribute,
         :manager_ref => [:name],
-        :association => :custom_attributes
+        :association => :custom_attributes,
       }
 
       attributes.merge!(extra_attributes)
     end
 
     def datacenters(extra_attributes = {})
       attributes = {
-        :model_class          => ::Datacenter,
-        :association          => :datacenters,
-        :attributes_blacklist => [:ems_children]
+        :model_class    => ::Datacenter,
+        :association    => :datacenters,
+        :builder_params => {
+          :ems_id => ->(persister) { persister.manager.id },
+        },
+      }
+
+      attributes.merge!(extra_attributes)
+    end
+
+    def vm_folders(extra_attributes = {})
```

Comment:
For the default infra manager collection we should leave this as ems_folders and if you need these in ovirt you can add them there